DESCRIPTION

```diff
@@ -40,6 +41,9 @@ Suggests:
     spelling,
     striprtf,
     xml2
+Remotes:
+    quanteda/quanteda@modularise,
+    quanteda/quanteda.textstats
```

Comment:
Thanks for the work on this, @kbenoit! I am happy with these changes. But could you clarify this part for me? From your description of the PR I had the impression that the changes should work with CRAN versions of these packages. And isn't the Remotes field only for packages not on CRAN yet? Thanks

Reply:
Sorry, you're totally right. I've just changed this to:
- remove the Remotes (**quanteda.textstats** is now on CRAN) 
- change the version requirement for **quanteda** back to what it was - it now works with both the existing 2.1.2 and the forthcoming, modularised pre-v3 (on GitHub)
- increment the version by 0.0.1 and change the release date.